index.js

```diff
@@ -1,17 +1,34 @@
+
 import isObservable from 'is-observable';
 import symbolObservable from 'symbol-observable';
 
-export default async function observableToPromise(value) {
+/**
+ * @template T
+ * @param {T extends unknown<T>} value
+ * @param {{maximumValues?: number}} options
+ * @returns {Promise<T[]>}
+ */
+export default async function observableToPromise(value, options) {
+	const {maximumValues = 0} = options || {};
```

Comment:
```suggestion
export default async function observableToPromise(value, {maximumValues = 0} = {}) {
```

Reply:
Looks better, Thank you for the suggestion.